theme/nav.html

```diff
@@ -47,6 +47,34 @@
                     {% include "subnavigation.html" %}
                 </div>
             {% endif %}
+            <div class="nav navbar-nav navbar-collapse visible-xs-block component-selector">
+                <h4 class="component-selector__title">Components</h4>
+                <h5 class="component-selector__subtitle">Find documentation…</h5>
+                <select class="form-control component-selector__control">
+                    <option value="/">Overview</option>
+                    <option value="tutorials">Tutorials</option>
+                    <optgroup label="Components">
```

Comment:
Is this list hardcoded to start, just so something is there before it loads the list from elsewhere?

Reply:
Hardcoded for testing and demonstration only.